.goreleaser.yml

```diff
@@ -32,11 +32,11 @@ brews:
     tap:
       owner: cirruslabs
       name: homebrew-cli
-    caveats: See the Github repository for more information
+    caveats: See the GitHub repository for more information
     homepage: https://github.com/cirruslabs/tart
     description: Run macOS VMs on Apple Silicon
     skip_upload: auto
     dependencies:
       - "cirruslabs/cli/softnet"
     custom_block: |
-      depends_on :macos => :monterey
+      depends_on :macos => :ventura
```

Comment:
Considering that the Ventura is still in beta until October, don't you think it's too early to drop Monterey support?

We might have multiple of features and bug fixes available in that time frame that will be of use for Monterey users.

[Dynamic](https://github.com/mhdhejazi/Dynamic) might be an option, though.

Reply:
But we have a bunch of exclusive features planned on... I was thinking to `ifdef` things around which will work for changes like this but the Linux support and folder sharing will be very hard to maintain IMO.

Another idea might be not to require Ventura but also don't use ifdef. Instead structure code so the codepaths of old features will continue working because of dynamic ducktyping. What do you think?

Reply:
See [0e2e74c](https://github.com/cirruslabs/tart/pull/175/commits/0e2e74cbc07a71b3d3457adf69b44f9675ab3d50)